### ISSUE TITLE
Update to 1.20 on macOS High Sierra Black Screen Instead of Workspace

### ISSUE THREAD

**Issue by TomReddish-MVC** [Microsoft/vscode#43624]
After updating to 1.20 from 1.19, when I launch vs code I get a window but with a black screen as if the workspace wont load.

- VSCode Version: 1.20
- OS Version:  macOS High Sierra 10.13.3 

Steps to Reproduce:

1. Launch VS Code

launching with "code --disable-extensions" does not fix the problem

running "code --verbose" I get the below output. It seems like it has something to do with my custom theme extension but I ran the same code with --disable-extensions and got the same output.

```
[main 6:15:57 PM] Starting VS Code
[main 6:15:57 PM] from: /Applications/Visual Studio Code.app/Contents/Resources/app
[main 6:15:57 PM] args: { _: [],
  help: false,
  h: false,
  version: false,
  v: false,
  wait: false,
  w: false,
  diff: false,
  d: false,
  add: false,
  a: false,
  goto: false,
  g: false,
  'new-window': false,
  n: false,
  'unity-launch': false,
  'reuse-window': false,
  r: false,
  'open-url': false,
  performance: false,
  p: false,
  'prof-startup': false,
  verbose: true,
  logExtensionHostCommunication: false,
  'disable-extensions': false,
  disableExtensions: false,
  'list-extensions': false,
  'show-versions': false,
  nolazy: false,
  issue: false,
  'skip-getting-started': false,
  'skip-release-notes': false,
  'sticky-quickopen': false,
  'disable-restore-windows': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-crash-reporter': false,
  'skip-add-to-recently-opened': false,
  status: false,
  s: false,
  'file-write': false,
  'file-chmod': false }
[main 6:15:57 PM] Resolving machine identifier...
[main 6:15:57 PM] Resolved machine identifier: b12f5ab3096f9664030e85ea7dca29a4f506f8e1211123160d930a3db316d491
[main 6:15:57 PM] update#setState idle
2018-02-13 18:15:57.130 Electron[1656:75241] *** WARNING: Textured window <AtomNSWindow: 0x7fbb41c78bc0> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.
[1656:0213/181558.090159:INFO:CONSOLE(1306)] "%cTRACE", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js (1306)
[1656:0213/181558.293296:INFO:CONSOLE(5)] "[uncaught exception]: TypeError: Cannot use 'in' operator to search for '[T-Redd]' in true", source: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/bootstrap/index.js (5)
[1656:0213/181558.293360:INFO:CONSOLE(5)] "TypeError: Cannot use 'in' operator to search for '[T-Redd]' in true
    at e.setCustomTokenColors (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3166:568)
    at new e (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:3173:679)
    at t.create (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:90:535)
    at e._createInstance (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1289:289)
    at e.createInstance (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1288:560)
    at e.initServiceCollection (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4694:51)
    at e.createContents (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4687:331)
    at e.open (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4695:415)
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4701:837
    at Object.g [as _notify] (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:140:868)", source: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/bootstrap/index.js 
```

Does this issue occur when all extensions are disabled?: Yes

**Comment by usernamehw** [comment]
Could you try to comment out `"editor.tokenColorCustomizations"` in your [Settings](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations)?

**Comment by TomReddish-MVC** [comment]
Can I do that externally? I cant edit any settings because the window is completely black

**Comment by usernamehw** [comment]
Well, yes. Open it with another editor.

**Comment by TomReddish-MVC** [comment]
hahaha should have clicked the link, wasn't sure if the settings were set with an internal DB to the app

**Comment by TomReddish-MVC** [comment]
Okay setting 
`"editor.tokenColorCustomizations = false"` 
fixed the problem.

**Comment by usernamehw** [comment]
Then you can close this issue as it is a duplicate of #43129 . It is fixed and will be out in the recovery build `1.20.1`.
